change/@ni-nimble-components-bcea55bf-ed59-4b11-b80b-6311749da460.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Oh it's subtle, here is a slowed down gif of it
![flickermenu](https://github.com/ni/nimble/assets/1588923/492f8e84-995f-4cfa-8e0b-bb440b3cab0c)

Reply:
Yep. So this is a result of using `display: none` instead of `visibility: hidden` to control the visibility of the dropdown. I've now reverted that change, removed the `IntersectionObserver`, and put back in the `requestAnimationFrame`. All of this along with the existing `loaded` event handler for the `anchoredRegion` still seems enough to make the integration into SLE use-case continue to work.

Reply:
If the `    .anchored-region[hidden] { visibility: hidden;}` can we still move it to the anchored region styles?
We should also have a comment that says we are intentionally not using the display helper.

```
    :host {
        ${'' /* Avoid using the `display` helper to customize hidden behavior }
        display: block;
        contain: layout;
        z-index: ${ZIndexLevels.zIndex1000};
    }
    :host[hidden] {
        visibility: hidden;
    }
```